app/tests/test_database_router.py

```diff
@@ -0,0 +1,29 @@
+import warnings
+from copy import deepcopy
+
+from bod.models import ContactOrganisation
+from provider.models import Provider
+
+from django.conf import settings
+from django.test import TestCase
+
+
+class DatabaseRouterTestCase(TestCase):
+
+    databases = {'default', 'bod'}
+
+    def test_database_routing(self):
+        self.assertEqual(ContactOrganisation.objects.db, 'bod')
+        self.assertEqual(Provider.objects.db, 'default')
+
+    def test_writing_to_bod_supported_within_tests(self):
+        ContactOrganisation.objects.create()
+        self.assertEqual(ContactOrganisation.objects.count(), 1)
+
+    def test_writing_to_bod_not_supported_outside_tests(self):
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", UserWarning)
```

Comment:
What warnings are suppressed here? Maybe worth a comment as it's not obvious to me.

Reply:
Ok, I see an explanatory comment in the most recent version. Thanks :+1: